lib/posix/clock.c

```diff
@@ -17,6 +18,19 @@
  */
 static struct timespec rt_clock_base;
 
+#ifdef CONFIG_RTC
+static struct device *clock_getrtc(void)
+{
+	static int isfirst = 1;
```

Comment:
variable not used. It could be used to fetch device only once. Now if get binding returns null then it will always be called only to use uptime as clock source.

Reply:
I removed it